ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerDefinitionExpression/FeatureLayerDefinitionExpression.h

```diff
@@ -34,26 +35,33 @@ class FeatureLayerDefinitionExpression : public QQuickItem
   Q_OBJECT
 
   Q_PROPERTY(bool layerInitialized READ layerInitialized NOTIFY layerInitializedChanged)
+  Q_PROPERTY(int currentFeatureCount READ currentFeatureCount NOTIFY currentFeatureCountChanged)
 
 public:
   explicit FeatureLayerDefinitionExpression(QQuickItem* parent = nullptr);
   ~FeatureLayerDefinitionExpression() override;
 
   void componentComplete() override;
   static void init();
-  Q_INVOKABLE void setDefExpression(QString whereClause);
+  Q_INVOKABLE void setDefExpression(const QString& whereClause);
+  Q_INVOKABLE void setDisplayFilter(const QString& whereClause);
+  void queryFeatureCountInCurrentExtent();
```

Comment:
🔧  optional but if this isn't called from QML code, it can be private.

Reply:
> I also noticed that one of the arcgisruntime.pri files changes permissions 100755 → 100644. Was that intentional?

I compared the .pri file permissions with arcgisruntime.pri in other samples and it seems to be the same as other instances (644). I wonder if that changes was picked up from when I merged with v.next. It doesn't seem to cause any issues